### ISSUE TITLE
Making start()/stop() non-blocking?

### ISSUE THREAD

**Issue by drapostolos** [drapostolos/rdp4j#3]
What's your opinion about making these methods non-blocking:
```java
DirectoryPollerBuilder::start();
DirectoryPoller::stop();
```
They are currently (rdp4j-0.0.8) blocking.

**Comment by drapostolos** [comment]
The above is a non-backward compatible change in behavior. So maybe a better idea is to add new methods instead (and keeping the old methods as is):
```java
DirectoryPollerBuilder::nonblockingStart();
DirectoryPoller::nonBlockingStop();
```

Comments?

**Comment by drapostolos** [comment]
A fix for this has now been pushed to master. See commit: 8e9b28c131ee449bb180d68471a126010970190c

Methods are called:

```java
DirectoryPollerBuilder::StartAsync();
DirectoryPoller::StopAsync();
```

**Comment by drapostolos** [comment]
Fixed in version 0.2.1
